examples/discord_to_desktop_notification.yaml

```diff
@@ -1,43 +1,33 @@
-apiVersion: v1
 kind: IOConfig
-module: mewbot.io.discord
-name: DiscordIO
-uuid: 1
+implementation: mewbot.io.discord.DiscordIO
+uuid: df56b38a-8499-4cec-95c1-b54192b4328a
 properties:
   token: "[token goes here]"
 
 ---
 
-apiVersion: v1
 kind: IOConfig
-module: mewbot.io.desktop_notification
-name: DesktopNotificationIO
-uuid: 2
+implementation: mewbot.io.desktop_notification.DesktopNotificationIO
+uuid: df56b38a-8499-4cec-95c1-b54192b4328a
```

Comment:
Ah! I had missed a trick, and you have a uuid verifier regex in the schema.

Would still recommend using different uuids for each component - to prevent (slightly) confusing examples. My worry is that people copying them would infer you _need_ to have the same uuid for each component in a config. Which, I believe, is not only not true but may lead to some internal confusion.

Reply:
Yes, I probably should -- at some point, we're going to start using these UUIDs for things, and I imagine that will break all the things.
I might change them to `aaaaaaaa-aaaa-4aaa-aaaa-00000000000[0-9]`?

Reply:
That sounds good to me - hard to mistake that for meaningful structure!